JS/app.js

```diff
@@ -138,37 +145,45 @@ new AdImage('bag pic', "img/bag.jpg");
 //==================================================
 //===================CHART===========================
 //===================================================
-var labelData = [];
-var chartData = [];
+
+function renderChart() {
+
+    var labelData = [];
+    var clickData = [];
+    var viewData = [];
+
+  for (var i= 0; i< AdImage.allImages.length; i++){
+    labelData.push(AdImage.allImages[i].name);
+    clickData.push(AdImage.allImages[i].clicked);
+    viewData.push(AdImage.allImages[i].views);
+    
+      }
+    //   var ctx = document.getElementById('whiteboard').getContext('2d');
+    
 
 var ctx = document.getElementById('whiteboard').getContext('2d');
+console.log(clickData, viewData);
```

Comment:
Great idea to console.log these arrays in order to see if they are full of data or not.